.travis.yml

```diff
@@ -20,8 +20,10 @@ env:
 
   matrix:
   - LINT_CHECK="1"
-  - TESTS="1" ODOO_REPO="OCA/OCB" MAKEPOT="1"
-  - TESTS="1" ODOO_REPO="odoo/odoo"
+  - TESTS="1" ODOO_REPO="OCA/OCB" MAKEPOT="1" EXCLUDE=hr_timesheet_task_required
```

Comment:
@mreficent why this is needed?

Reply:
Because `hr_timesheet_task_required` is installed before `hr_timesheet_sheet_no_negative`, and thus in the `hr_timesheet_sheet_no_negative` tests the task is required, and I don't want to put task in the tests.

Reply:
But this would effectively double the CI time

Reply:
Maybe let's just add context flag to exclude that check?

Reply:
done the context flag